egs/wsj/s5/utils/data/perturb_data_dir_volume.sh

```diff
@@ -52,18 +52,19 @@ for line in sys.stdin.readlines():
   parts = line.strip().split()
   if line.strip()[-1] == '|':
     if re.search('sox --vol', ' '.join(parts[-11:])):
-      print 'true'
+      print("True")
```

Comment:
please don't change the capitalization of these things.  The standard things being referred to here are /usr/bin/false and /usr/bin/true, the upper-cased versions don't work on Debian linux at least.

Reply:
for me the lower-cased true/false was not working. Only, True/False was working. So I write True/False to volume_perturb_done. 
Traceback (most recent call last):
  File "<string>", line 10, in <module>
NameError: name 'true' is not defined

Reply:
That's because you also changed the single quotes to double quotes.  It's part of a double-quoted string, so double-quotes disappear.

Reply:
Use of single quotes helped. I have modified the PR